src/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Todo App: Second Gen</title>
+</head>
+<body>
+    <section class="main">
+        <div class="list-box">
+            <div class="title">
+                <h3>Today's To Do</h3>
+                <button class="refresh"></button>
+            </div>
+            <input id="new-task" type="text" value="" placeholder="Add to your list...">
+            <ul id="list" class="list">
```

Comment:
[ optional ] Assign other class if it's occuped by other class or id add a <span ="class">

Reply:
Thank you for your suggestion!